README.md

```diff
@@ -244,6 +244,26 @@ To lint the Python code, run:
 make lint
 ```
 
+#### Django Migration Linter
+
+[Django Migration Linter](https://github.com/3YOURMIND/django-migration-linter) for linting migrations files in the project.
+
+Before you run the commands below to lint the migration files, make sure you have shelled into the docker web container. As the migration linter will need access to the postresql database.
+
+To lint the django migration files:
+
+```bash
+make lint-migrations
+```
+
+To lint the django migration files and view the output in a more readable format:
+
+```bash
+make lint-migrations-custom
+```
```

Comment:
This can go too 😄

Reply:
added 🙏 [c1083d1](https://github.com/ONSdigital/dis-wagtail/pull/43/commits/c1083d15710f5abb98e8bbd2cf3803d04b24c1a7)